Stomatogastric ganglion cell types of crabs, lobsters

stevevanhooser opened the pull request:
With @andreagaede, added 8 cell types from the stomatogastric ganglion of crabs and lobsters (thought to be homologous neurons).

It references a relatively new UBERON node: UBERON:8910001 ! stomatogastric ganglion

It views well in Protoge

stevevanhooser commented:
Not sure which reviewer to request (apparently I cannot suggest one), but we discussed these changes with @gouttegd and @rays22 at the November meeting.

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,96 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+AnnotationAssertion(terms:contributor obo:CL_4070010 <https://orcid.org/0000-0002-1909-7004>)
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon:6712))
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon:6752))
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON:8910001))
```
Side note (more intended for other reviewers): `UBERON:8910001` has only been added to Uberon last month, and there has been no new Uberon release since then.  We will need to wait for a Uberon release and then refresh the imports.

gouttegd reviewed (changes requested):
Please see the inline comments below. I only commented on the first new term, but the comments apply, _mutatis mutandis_, to all the proposed new terms.

It is especially important to fix the incorrect use of `EquivalentClasses` and of the `in taxon` relationship.

dosumis commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,96 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+EquivalentClasses(obo:CL_4070010 ObjectIntersectionOf(obo:CL_0000540 obo:CL_0000100))
+AnnotationAssertion(terms:contributor obo:CL_4070010 <https://orcid.org/0000-0002-1909-7004>)
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon:6712))
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon:6752))
```
I think a general 'in taxon' for crustacea would be useful/ This would help us to automatically filter out from views used to drive resources for vertebrates.  Note - this does not assert that these neurons are present in all Crustacea it only limits them to Crustacea.  You might also want to add 'present in taxon' for taxa where these are confirmed to be present.

stevevanhooser replied:
thanks, I understand now, fixed!

stevevanhooser commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,96 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+
+AnnotationAssertion(rdfs:label obo:CL_4070010 "gastric mill neuron")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4070010 "GM")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI: 10.1007/978-3-642-71516-7_2") obo:IAO_0000115 obo:CL_4070010 "Medial tooth protraction: The medial tooth is moved forward by contraction of extrinsic muscles innervated by the GM neuron")
```
Thanks, doi now lowercase, spaces removed from doi: and PMID: 

Definitions updated

stevevanhooser commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,96 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+AnnotationAssertion(rdfs:label obo:CL_4070010 "gastric mill neuron")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4070010 "GM")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI: 10.1007/978-3-642-71516-7_2") obo:IAO_0000115 obo:CL_4070010 "Medial tooth protraction: The medial tooth is moved forward by contraction of extrinsic muscles innervated by the GM neuron")
+EquivalentClasses(obo:CL_4070010 ObjectIntersectionOf(obo:CL_0000540 obo:CL_0000100))
```
Thanks, I understand now, fixed

stevevanhooser commented:
Fixes made, looks good to me in Protege, this note from @gouttegd remains relevant:

> Side note (more intended for other reviewers): UBERON:8910001 has only been added to Uberon last month, and there has been no new Uberon release since then. We will need to wait for a Uberon release and then refresh the imports.

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+
+AnnotationAssertion(rdfs:label obo:CL_4070010 "gastric mill neuron")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4070010 "GM")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1007/978-3-642-71516-7_2") obo:IAO_0000115 obo:CL_4070010 "A motor neuron that moves the medial tooth forward")
```
Definition should be proper sentences, that end with a final dot.

stevevanhooser replied:
👍

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+
+# Class: obo:CL_4070013 (ventricular dilator motor neurons)
+
+AnnotationAssertion(rdfs:label obo:CL_4070013 "ventricular dilator motor neurons")
```
The label should always be singular.

stevevanhooser replied:
👍

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+
+AnnotationAssertion(rdfs:label obo:CL_4070014 "pyloric motor neuron")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4070014 "PY")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1007/978-3-642-71516-7_2") Annotation(oboInOwl:hasDbXref "PMID:6296329") obo:IAO_0000115 obo:CL_4070014 "A motor neuron that controls pyloric filter movements")
```
`pyloric motor neuron` (here) and `lateral pyloric motor neuron` (further below) have exactly the same definition. This is not allowed in CL. Consider adding something to the definition of `lateral pyloric motor neuron` to clearly distinguish that term from `pyloric motor neuron`.

stevevanhooser replied:
Thanks, this is a tricky one. They innervate the same structure but antagonistic muscles

Changed to `"A motor neuron that controls pyloric filter movements; innervates muscles that are antagonized by the lateral pyloric neurons." 

and `"A motor neuron that controls pyloric filter movements; innervates muscles that are antagonized by the pyloric motor neurons.`

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+
+# Class: obo:CL_4070015 (pyloric dilator)
+
+AnnotationAssertion(rdfs:label obo:CL_4070015 "pyloric dilator")
```
I would suggest naming this class “pyloric dilator **neuron**”.

stevevanhooser replied:
👍

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+
+# Class: obo:CL_4070016 (anterior burster)
+
+AnnotationAssertion(rdfs:label obo:CL_4070016 "anterior burster")
```
Likewise, I would suggest naming this “anterior burster neuron“.

stevevanhooser replied:
👍

gouttegd reviewed (changes requested):
Thank you for your reactivity, much appreciated!

Out of the following comments, the critical one is the one about the duplicated definitions of CL:4070014 and CL:4070017.

With the other comments, we are entering into “nitpicking“ territory, which is good as it means that the most important issues have been addressed.

stevevanhooser commented:
Thanks also for your quick responses. Updated

dosumis commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+AnnotationAssertion(terms:contributor obo:CL_4070013 <https://orcid.org/0000-0002-1909-7004>)
+SubClassOf(obo:CL_4070013 obo:CL_0000100)
+SubClassOf(obo:CL_4070013 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon:6657))
+SubClassOf(obo:CL_4070013 ObjectSomeValuesFrom(obo:RO_0002175 obo:NCBITaxon:6712))
```
present in taxon should be recorded as an annotation property axiom (there is some under-the-hod conversion to the rather weak logical axioms that this represents).

gouttegd replied:
```
AnnotationAssertion(obo:RO_0002175 obo:CL4070013 obo:NCBITaxon:6712)
```

Are you editing the ontology “manually” (with a text editor)?

stevevanhooser replied:
Yep, using a text editor!
What do you all use? I was frightened by the many buttons of Protege but I suppose it is the right way?

Regarding the change here, I will add


```
AnnotationAssertion(obo:RO_0002175 obo:** obo:NCBITaxon:6712)
AnnotationAssertion(obo:RO_0002175 obo:** obo:NCBITaxon:6752)
```

but 

Q1: do I also remove 

```
SubClassOf(obo:* ObjectSomeValuesFrom(obo:RO_0002175 obo:NCBITaxon:6712))
SubClassOf(obo:* ObjectSomeValuesFrom(obo:RO_0002175 obo:NCBITaxon:6752))
```

and Q2:

I leave this alone?

`SubClassOf(obo:CL_4070013 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon:6657))`

gouttegd replied:
> What do you all use?

AFAIK most (all?) CL editors are using Protégé Desktop. Not that this is an obligation -- there is nothing wrong with editing manually as long as you know how to do it.

> Q1: do I also remove

Yes.

> Q2: I leave this alone?

Yes (but see next remark below). `RO:0002162` is an object property, so a `SubClassOf` axiom is correct.

I edited my previous comment to add that, in all those axioms, your NCBITaxon IDs are incorrect. They should be of the form `obo:NCBITaxon_xxxx`, _not_ `obo:NCBITaxon:xxxx`. This must be changed in all axioms.

stevevanhooser replied:
NCBITaxon fixed

Q1 and Q2 resolved

stevevanhooser replied:
@gouttegd I'm missing the remark related to "RO:0002162 is an object property, so a SubClassOf axiom is correct."

gouttegd replied:
I was referring to the next paragraph about the incorrect NCBITaxon ID -- what I meant was “leave this axiom alone, but do fix the incorrect ID within it”.

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4070010 "GM")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1007/978-3-642-71516-7_2") obo:IAO_0000115 obo:CL_4070010 "A motor neuron that moves the medial tooth forward")
+AnnotationAssertion(terms:contributor obo:CL_4070010 <https://orcid.org/0000-0002-1909-7004>)
+AnnotationAssertion(obo:RO_0002175 obo:CL_4070010 obo:NCBITaxon:6712)
```
This is again an invalid ID. It should be `obo:NCBITaxon_6712`.

stevevanhooser replied:
Ah, it snuck back in there when I did the Annotation (order of operations! I fixed the NCBI: first)

Thanks, and sorry!

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33507,6 +33515,102 @@ SubClassOf(obo:CL_4052049 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_001576
+AnnotationAssertion(obo:RO_0002175 obo:CL_4070010 obo:NCBITaxon:6752)
+SubClassOf(obo:CL_4070010 obo:CL_0000100)
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_6657))
+SubClassOf(obo:CL_4070010 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON:8910001))
```
Add I had also missed earlier that this is also an invalid Uberon ID. It should be `obo:UBERON_8910001`.

It might be useful to elaborate a bit on why we are sometimes using a `_` and sometimes a `:` in IDs (e.g, why do I speak of `UBERON:8910001` whereas here I say the identifier should be `obo:UBERON_8910001`).

As a general rule, all IDs in OBO ontologies are of the form `http://purl.obolibrary.org/obo/ONT_XXXX`, where `ONT` is the short name of the ontology (e.g. `CL`, `UBERON`, `NCBITaxon`, etc.) and `XXXX` is the numerical part.

But in many places (including in CL’s edit file), we write identifiers in a _short form_ (aka _CURIE form_), made of a _prefix name_, followed by a colon, followed by a _local name_.

What the short form looks like exactly depends on the prefix name chosen. For `http://purl.obolibrary.org/obo/UBERON_8910001` for example, it can be written as

* `UBERON:8910001` <- here the `UBERON` prefix name stands for `http://purl.obolibrary.org/obo/UBERON_`;
* `obo:UBERON_8910001` <- here the `obo` prefix name stands for `http://purl.obolibrary.org/obo/` 

Both form would expand into `http://purl.obolibrary.org/obo/UBERON_8910001` and would therefore be correct (though in CL’s edit file we only use the second form).

But a form like `obo:UBERON:8910001` cannot be correct. This would expand into `http://purl.obolibrary.org/obo/UBERON:8910001`, which is _not_ a valid Uberon identifier.

The same logic applies to NCBITaxon IDs. They can be written _either_ as `NCBITaxon:XXXX`, or as `obo:NCBITaxon_XXXX` (though again, in CL’s edit file only that last form is used), but _not_ as `obo:NCBITaxon:XXXX`.

I hope this clarifies things a bit.

stevevanhooser replied:
Thanks! Got it, in OWL code, always ONT_NODE, whereas in notes and some other places (obo files) it is ONT:NODE.

Thanks!

gouttegd reviewed (approved):
This looks good to me.

As previously noted though, we’ll need to wait for UBERON:8910001 to become available in the next Uberon release.

stevevanhooser commented:
Thanks for your help @gouttegd! 

Are these releases at regular intervals or just as the developers are ready?

Thanks again
Steve

gouttegd commented:
We try to make one release every 6 to 8 weeks. The last Uberon release was in late November, so I’d expect the next one to happen before the end of the month. No promises, though.

aleixpuigb commented:
> This looks good to me.
> 
> As previously noted though, we’ll need to wait for UBERON:8910001 to become available in the next Uberon release.

I can run an Uberon release soon (early next week to give enough time to close PRs). I will make the announcement.

stevevanhooser commented:
Did an Uberon release happen? It seems to be updated on https://www.ebi.ac.uk/ols4/ontologies/uberon

If so, then perhaps this can be merged?

aleixpuigb commented:
Hi @stevevanhooser, there is a new Uberon release. Do you need help importing the term from Uberon?

stevevanhooser commented:
Yes please. I don't quite follow the documentation there.

aleixpuigb commented:
You need to add UBERON:8910001 in the uberon imports txt file https://github.com/obophenotype/cell-ontology/blob/master/src/ontology/imports/uberon_terms.txt . Once that is done, open the docker and in the terminal navigate to src/ontology. Then run:
```
sh run.sh make imports/merged_import.owl   
```
Once that is done you should be able to see the term in protégé inside CL. Please let me know if there are difficulties and I can assist you.

aleixpuigb commented:
I have tried to push the commit with the Uberon term imported, but I don't have permission to do it.

stevevanhooser commented:
Thanks @aleixpuigb. I sent an invitation to my repo if that's what you need access to. I am writing a grant this week and did not have a chance to get into learning about docker. 

Thanks

aleixpuigb commented:
No problem!

dosumis commented:
Looks like this has been ready to go for over a month but there are now some conflicts?  I suspect due to updates to imports in branch.

How can we get this resolved quickly to get @stevevanhooser 's rather awesome work into our next release?

gouttegd commented:
I have fixed the conflicts in the https://github.com/obophenotype/cell-ontology/tree/stomatogastric_ganglion_crab_lobster_fixed branch.

Since _this_ PR originates from another repository, I cannot push my fixes directly here. Instead, we have two solutions:

(A) I open a PR with the conflict resolution on the [original repository](https://github.com/Waltham-Data-Science/cell-ontology/tree/stomatogastric_ganglion_crab_lobster) on the Waltham Data Science account. From there, @stevevanhooser can merge it, which will automatically be reflected on _this_ PR, which we’ll then be able to merge.

(B) We discard this PR, and I open a new one containing both the original work and the conflict resolution.

(A) is a slightly more complicated approach, but it would have the benefit that the PR that is ultimately merged would retain all the discussion that we have been having here.

(B) is more straightforward, but it would appear as a completely new PR completely unrelated to this one.

stevevanhooser commented:
I can accept! A is fine. via mobileOn Mar 12, 2025, at 2:38 PM, Damien Goutte-Gattat ***@***.***> wrote:﻿
  I have fixed the conflicts in the https://github.com/obophenotype/cell-ontology/tree/stomatogastric_ganglion_crab_lobster_fixed [github.com] branch.
Since this PR originates from another repository, I cannot push my fixes directly here. Instead, we have two solutions:
(A) I open a PR with the conflict resolution on the original repository [github.com] on the Waltham Data Science account. From there, @stevevanhooser [github.com] can merge it, which will automatically be reflected on this PR, which we’ll then be able to merge.
(B) We discard this PR, and I open a new one containing both the original work and the conflict resolution.
(A) is a slightly more complicated approach, but it would have the benefit that the PR that is ultimately merged would retain all the discussion that we have been having here.
(B) is more straightforward, but it would appear as a completely new PR completely unrelated to this one.—Reply to this email directly, view it on GitHub [github.com], or unsubscribe [github.com].You are receiving this because you were mentioned.Message ID: ***@***.***>



gouttegd left a comment (obophenotype/cell-ontology#2876) [github.com]
I have fixed the conflicts in the https://github.com/obophenotype/cell-ontology/tree/stomatogastric_ganglion_crab_lobster_fixed [github.com] branch.
Since this PR originates from another repository, I cannot push my fixes directly here. Instead, we have two solutions:
(A) I open a PR with the conflict resolution on the original repository [github.com] on the Waltham Data Science account. From there, @stevevanhooser [github.com] can merge it, which will automatically be reflected on this PR, which we’ll then be able to merge.
(B) We discard this PR, and I open a new one containing both the original work and the conflict resolution.
(A) is a slightly more complicated approach, but it would have the benefit that the PR that is ultimately merged would retain all the discussion that we have been having here.
(B) is more straightforward, but it would appear as a completely new PR completely unrelated to this one.

—Reply to this email directly, view it on GitHub [github.com], or unsubscribe [github.com].You are receiving this because you were mentioned.Message ID: ***@***.***>

gouttegd commented on src/ontology/cl-edit.owl:
```diff
@@ -33756,6 +33764,17 @@ AnnotationAssertion(rdfs:label obo:CL_4052042 "tuft cell of urethra")
 EquivalentClasses(obo:CL_4052042 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002325)))
 SubClassOf(obo:CL_4052042 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_1904320))
 
+# Class: obo:CL_4052045 (steroidogenic stromal cell of ovary)
```
For other reviewers: this change here, which is seemingly unrelated to the rest of the PR, is simply due to the fact that I re-serialized the -edit file as part of the conflict resolution process. That class had somehow been written out-of-order (after the CL:4052027 class) at some point in the master branch, so now the re-serialization put it back at the right place. Everything’s normal.

gouttegd commented:
Because the PR was not merged soon enough after I fixed the conflicts 2 weeks ago, it is now _again_ in a conflicted state and _cannot_ be merged.

So, @stevevanhooser, we need to redo the same thing as last time. I’ve fixed the conflict locally and pushed them to [a PR on your repo](https://github.com/Waltham-Data-Science/cell-ontology/pull/2). Accept that PR over there and we can resume here. Sorry about that.

stevevanhooser commented:
Thank you, merged!

> On Mar 26, 2025, at 2:36 PM, Damien Goutte-Gattat ***@***.***> wrote:
> 
> 
> gouttegd
>  left a comment 
> (obophenotype/cell-ontology#2876) [github.com]
> Because the PR was not merged soon enough after I fixed the conflicts 2 weeks ago, it is now again in a conflicted state and cannot be merged.
> 
> So, @stevevanhooser [github.com], we need to redo the same thing as last time. I’ve fixed the conflict locally and pushed them to a PR on your repo [github.com]. Accept that PR over there and we can resume here. Sorry about that.
> 
> —
> Reply to this email directly, view it on GitHub [github.com], or unsubscribe [github.com].
> You are receiving this because you were mentioned.
> 
>  <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/stevevanhooser__;!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJo09US1s$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw2nXW_Z5SHzQzjg4WA37Wn0> <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/Waltham-Data-Science/cell-ontology/pull/2__;!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJon1Xd0M$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw0ERHdAZvVIHi6LN56qP4HV> <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/obophenotype/cell-ontology/pull/2876*issuecomment-2755418458__;Iw!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJMSHEYFk$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw2Kd1ZgD-rRqNpgS_4is3VH> <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AAOOPDKWALRK2EPILOECM232WLQQJAVCNFSM6AAAAABUWXISTKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJVGQYTQNBVHA__;!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJb2NsKMw$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw16vbvGty8U5HRgHOew0_nm>
> 
> gouttegd
>  left a comment 
> (obophenotype/cell-ontology#2876) [github.com]
>  <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/obophenotype/cell-ontology/pull/2876*issuecomment-2755418458__;Iw!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJMSHEYFk$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw2Kd1ZgD-rRqNpgS_4is3VH>
> Because the PR was not merged soon enough after I fixed the conflicts 2 weeks ago, it is now again in a conflicted state and cannot be merged.
> 
> So, @stevevanhooser [github.com] <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/stevevanhooser__;!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJo09US1s$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw2nXW_Z5SHzQzjg4WA37Wn0>, we need to redo the same thing as last time. I’ve fixed the conflict locally and pushed them to a PR on your repo [github.com] <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/Waltham-Data-Science/cell-ontology/pull/2__;!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJon1Xd0M$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw0ERHdAZvVIHi6LN56qP4HV>. Accept that PR over there and we can resume here. Sorry about that.
> 
> —
> Reply to this email directly, view it on GitHub [github.com] <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/obophenotype/cell-ontology/pull/2876*issuecomment-2755418458__;Iw!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJMSHEYFk$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw2Kd1ZgD-rRqNpgS_4is3VH>, or unsubscribe [github.com] <https://www.google.com/url?q=https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AAOOPDKWALRK2EPILOECM232WLQQJAVCNFSM6AAAAABUWXISTKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJVGQYTQNBVHA__;!!DaRZpAeNFA!fRtBzC3eP3U3HvAfbSPTs61FYRDWkrV1TXHPvGUsGQVdv6J2Nj38mmOpX7loZgl7OSFmN-ly7fu5BCOfFsIJb2NsKMw$&source=gmail-imap&ust=1743618968000000&usg=AOvVaw16vbvGty8U5HRgHOew0_nm>.
> You are receiving this because you were mentioned.
>

gouttegd commented:
@dosumis Since you requested changes in your last review (changes which have by now been done), your explicit approval is required before this PR can be merged.

dosumis commented:
#gogoeditdiff
